test/gurka/test_phonemes.cc

```diff
@@ -1150,16 +1130,15 @@ TEST(Standalone, PhonemesWithAltandDirection) {
   {
     GraphId node_id = BD_edge->endnode();
     auto tile = graph_reader.GetGraphTile(node_id);
-    const NodeInfo* node_info = tile->node(node_id);
+    [[maybe_unused]] const NodeInfo* node_info = tile->node(node_id);
```

Comment:
same here?

Reply:
yup, the variable is not used (assigned values but never used), so I will remove the variable but keep the operation.